spec/helpers/user_activity_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe UserActivityHelper, organization_workspace: :test do
+  helper UserActivityHelper
+
+  let(:organization) { Organization.current }
+
+  before do
+    # Set today date to Wednesday to test week always starts on Monday
+    allow(Date).to receive(:today).and_return(Date.new(2021, 2, 10))
+    allow_any_instance_of(UserActivityHelper).to receive(:min_week).and_return(8.weeks.ago(Date.today))
```

Comment:
Writing this comment to up the odds I'll remember to delete these mocks and the whole `date_and_time.rb` file when we upgrade to Rails 5.2